wsclient/src/main/java/org/hpccsystems/ws/client/platform/DropZone.java

```diff
@@ -49,6 +50,35 @@ public String getName()
         return info.getName();
     }
 
+    public String[] getConfNetAddresses()
```

Comment:
let's go ahead and expand the name so there's no doubt about what it does (getConfiguredNetAddresses), also let's add a method comment.
In the comment we can distinguish this method from getNetAddresses